<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alert::Internal::AlertItemEditorWidget</class>
 <widget class="QWidget" name="Alert::Internal::AlertItemEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QTextEdit" name="alertDecsr"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="alertDescription_Label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="alertCategory_Label">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="alertCategory"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="alertLabel"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="alertLabel_Label">
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="iconLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab_type">
      <attribute name="title">
       <string>Type</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QLabel" name="priority_Label">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="priority"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="viewType_Label">
         <property name="text">
          <string>View type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="viewType"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="contentType_Label">
         <property name="text">
          <string>Content type</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="contentType"/>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="overrideRequiresUserComment">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>override requires user comment</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="override_Label">
         <property name="text">
          <string>User override</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="alertRelated_Label">
         <property name="text">
          <string>Related to</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="relatedTo"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_timing">
      <attribute name="title">
       <string>Timing</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" colspan="3">
        <widget class="Alert::AlertItemTimingEditorWidget" name="timingEditor" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_scripts">
      <attribute name="title">
       <string>Script</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="Alert::Internal::AlertItemScriptEditor" name="scriptEditor" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_css">
      <attribute name="title">
       <string>CSS</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="css"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>CSS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_xml">
      <attribute name="title">
       <string>Extra XML content</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="xml"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Alert::AlertItemTimingEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">alertplugin/alertitemtimingeditorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Alert::Internal::AlertItemScriptEditor</class>
   <extends>QWidget</extends>
   <header location="global">alertplugin/alertitemscripteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>alertLabel</tabstop>
  <tabstop>alertCategory</tabstop>
  <tabstop>alertDecsr</tabstop>
  <tabstop>iconLineEdit</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>relatedTo</tabstop>
  <tabstop>priority</tabstop>
  <tabstop>viewType</tabstop>
  <tabstop>contentType</tabstop>
  <tabstop>overrideRequiresUserComment</tabstop>
  <tabstop>css</tabstop>
  <tabstop>xml</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
